internal/checker/terraform/rules/ibm/app_id_management/tfibm_appid_action_url_https_protocol.rego

```diff
@@ -0,0 +1,49 @@
+# METADATA
+# title: "Verify HTTPS protocol in action URL"
+# description: "This policy validates that the 'url' in the 'ibm_appid_action_url' resource uses 'https'. It checks if the URL is using a secure protocol."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/appid_action_url
+# custom:
+#   id: CB_TFIBM_60
```

Comment:
Please use our proper format: `CB_TFIBM_###`. 
If you are using this for your previous or recent PR, I advise you to fix it."